src/main/resources/public/css/stylesheet.css

```diff
@@ -55,7 +45,35 @@ span {
     margin-right: 2em;
 }
 
-html, body {
-    height:100vh;
+html,
+body {
+    height: 100vh;
     color: #3b3d44;
-}
\ No newline at end of file
+}
+
+input, textarea {
+    width: 100%;
+    margin: 8px 0;
+}
+
+input[type=text], textarea{
+    padding: 12px 20px;
+    display: inline-block;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+  
+  input[type=number]{
```

Comment:
why do we not want padding for type=number inputs? Looking at the page most input fields seem to be the same height but then you get to Inventory and Price and they are about half the height

Reply:
when the type=number are padded, they go offscreen unlike the other ones.

Reply:
also input[type=number{ => should have a space: input[type=number] {

Reply:
done